src/global/RuntimeParameters.h

```diff
@@ -65,6 +65,10 @@ inline auto& RuntimeParameters() {
         Bool<"zero-cost-estimate-for-cached-subtree">{false},
         // Maximum size for the body of requests that the server will process.
         MemorySizeParameter<"request-body-limit">{100_MB},
+        // SERVICE operations are not cached by default, but can be enabled
+        // which has the downside that the sibling optimization where VALUES are
+        // dynamically pushed into `SERVICE` can no longer be used.
```

Comment:
```suggestion
        // which has the downside that the sibling optimization where VALUES are
        // dynamically pushed into `SERVICE` is no longer used.
```
This second part is missing in the commandline interface option.